webapp/app.py

```diff
@@ -101,3 +103,41 @@ def get_users(username: str):
         return jsonify(list(users))
     else:
         return jsonify({"error": "Failed to fetch users"}), 500
+
+
+@app.route('/set-owners', methods=['POST'])
```

Comment:
I suggest calling it "/set-owner" since we only set one owner for a give webpage